Sources/SwiftDocC/Model/Rendering/Tutorial/References/DownloadReference.swift

```diff
@@ -39,25 +53,39 @@ public struct DownloadReference: RenderReference, URLReference {
     public init(identifier: RenderReferenceIdentifier, renderURL url: URL, sha512Checksum: String?) {
         self.identifier = identifier
         self.url = url
-        self.sha512Checksum = sha512Checksum
+        self.checksum = sha512Checksum
     }
     
     enum CodingKeys: String, CodingKey {
         case type
         case identifier
         case url
-        case sha512Checksum = "checksum"
+        case checksum
     }
     
     public func encode(to encoder: Encoder) throws {
         var container = encoder.container(keyedBy: CodingKeys.self)
         try container.encode(type.rawValue, forKey: .type)
         try container.encode(identifier, forKey: .identifier)
-        try container.encodeIfPresent(sha512Checksum, forKey: .sha512Checksum)
+        try container.encodeIfPresent(checksum, forKey: .checksum)
         
         // Render URL
         try container.encode(renderURL(for: url), forKey: .url)
     }
+
+    public init(from decoder: Decoder) throws {
```

Comment:
Do we need a customer decoder implementation?

Reply:
Xcode balked at me when i initially added the computed property, though i bet that may have been because the coding keys were still pointing to the computed property instead of the stored one. Let me see if i can ditch it now that i've renamed the coding key.

Reply:
Since the names of the properties now match the JSON keys, we also don't need the custom coding keys anymore either.

Reply:
You're right, i was able to drop both the decoder implementation and the custom coding keys with no problem.